### PR TITLE
Cart changes:

### PR DESCRIPTION
**Pull request by galpinto25**
- added new class CartAdapter for the products of the cart
- added new view product_card_view
- fixed bug with radio buttons
- added recycler view in the CartActivity with details and edit/delete buttons

### CODE REVIEW

File: .idea/dictionaries/galium31.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <component name="ProjectDictionaryState">
   <dictionary name="galium31">
     <words>
+      <w>pizzz</w>
```
Comment (eilonasher) [inline .idea/dictionaries/galium31.xml:4]: design changes, nothing from me to add

File: app/src/main/java/com/example/sabonit/CartActivity.java [hunk 2]
```diff
@@ -6,27 +6,65 @@
...
+    private ArrayList<Order> orders;
+    private FirebaseFirestore db;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_cart);
         cartDescription = findViewById(R.id.items_list);
-        printCartDescription();
+        cart = Account.getCurrentAccount().getCart();
```
Comment (eilonasher) [inline app/src/main/java/com/example/sabonit/CartActivity.java:32]: it's great that you know how to use it, im learning from you

File: app/src/main/java/com/example/sabonit/CartAdapter.java [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+import java.util.List;
+
+//******************pizza adapter*******************************/
+
+
+/**
+ * A class which represents an adapter of Pizza objects, which extends
+ * the RecyclerView.Adapter<CartAdapter.ViewHolder> class.
+ */
+public class CartAdapter extends RecyclerView.Adapter<CartAdapter.ViewHolder> {
```
Comment (eilonasher) [inline app/src/main/java/com/example/sabonit/CartAdapter.java:34]: in the cart activity screen, all the functionalty happens from here?

**Review by eilonasher — COMMENTED** [review]
from me its yes for merging, good changes

File: app/src/main/java/com/example/sabonit/CartAdapter.java (hunk 3 above)
Comment (galpinto25) [reply app/src/main/java/com/example/sabonit/CartAdapter.java:34]: Yes, all the functionality (of the CartActivity) happens by the CartAdapter class which extends the RecyclerView.Adapter<CartAdapter.ViewHolder> class

**Pull request merged**